x/tokenfactory/types/params.go

```diff
@@ -1,40 +1,40 @@
 package types
 
 import (
-	fmt "fmt"
+	"fmt"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
-
-	"github.com/neutron-org/neutron/app/params"
 )
 
 // Parameter store keys.
 var (
-	KeyDenomCreationFee              = []byte("DenomCreationFee")
-	DefaultNeutronDenom              = params.DefaultDenom
-	DefaultFeeAmount           int64 = 1_000_000
-	KeyFeeCollectorAddress           = []byte("FeeCollectorAddress")
-	DefaultFeeCollectorAddress       = ""
+	KeyDenomCreationFee        = []byte("DenomCreationFee")
+	KeyDenomCreationGasConsume = []byte("DenomCreationGasConsume")
```

Comment:
Since we do not use paramsKeeper anymore, its safe to remove these keys

Reply:
We argeed to set both params to `0`